roles/ohpc_add_rabbitmq_agents/templates/agent.j2

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description={{ item.desc }}
+After=rabbitmq-server.service
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User=centos
+WorkingDirectory={{ rabbitmq_agents_loc }}
+Environment="PATH={{ rabbitmq_agents_loc }}/venv/bin"
+ExecStart={{ rabbitmq_agents_loc }}/venv/bin/python {{ item.name }}_create.py
```

Comment:
This naming convention suits with `_create` for the agents in dev env but when it comes to prod it doesn't . So we might have to change this to {{ item.name }}.py

Reply:
I fixed this in [039413e](https://github.com/jprorama/CRI_XCBC/commit/039413e118977d9f46500d164623e68938455893)